test/addBinding.test.ts

```diff
@@ -45,7 +46,7 @@ suite('Add Binding', () => {
     });
 
     test('Tree', async () => {
-        const treeItem: AzExtTreeItem | undefined = await ext.tree.findTreeItem(`/localProjecttestWorkspace/functions/${functionName}`, createTestActionContext());
+        const treeItem: AzExtTreeItem | undefined = await ext.tree.findTreeItem(`/localProject0/functions/${functionName}`, createTestActionContext());
```

Comment:
It's because that's the first folder in "test.code-workspace". Each parallel test in a suite will grab the next available workspace folder. I gave the folders generic names (0,1,2,3...) because for parallel tests you never know which test will grab which folder and so the names don't mean anything. In this case, though, this suite doesn't run any parallel tests and it only uses one folder, so I know it'll use the first one.